src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -130,7 +278,7 @@ export default class FileField extends React.Component {
           <ul className="schemaform-file-list">
             {files.map((file, index) => {
               const errors = _.get([index, '__errors'], errorSchema) || [];
-              const hasErrors = errors.length > 0;
+              const hasErrors = errors.length > 0 || file.errorMessage;
```

Comment:
Instead of modifying `hasErrors` which is downstream of the error source, could we make sure that `file.errorMessage` gets included in the `errors`?